client/signup/steps/user/index.jsx

```diff
@@ -466,6 +466,7 @@ export class UserStep extends Component {
 					}
 					horizontal={ isReskinned }
 					isReskinned={ isReskinned }
+					displayUsernameInput={ this.props.stepName === 'user' && ! this.props.isReskinned }
```

Comment:
Instead of these two checks, shouldn't we check for `account` flowName? We want to show the username input only for [this signup flow](https://github.com/Automattic/wp-calypso/blob/trunk/client/signup/config/flows-pure.js#L22-L29).

Reply:
Great find @razvanpapadopol! I updated the logic in 5fa72ff82471097687fb8e31ae6dca1b3121e8ff.